ynput/OpenPype
# RuntimeError with Click on deadline publish

## Changelog Description
I changed Click to version 8.0 instead of 7.1.2 to solve this error:
```
2023-05-30 16:16:51:  0: STDOUT: Traceback (most recent call last):
2023-05-30 16:16:51:  0: STDOUT:   File "start.py", line 1126, in boot
2023-05-30 16:16:51:  0: STDOUT:   File "/prod/softprod/apps/openpype/LINUX/3.15/dependencies/click/core.py", line 829, in __call__
2023-05-30 16:16:51:  0: STDOUT:     return self.main(*args, **kwargs)
2023-05-30 16:16:51:  0: STDOUT:   File "/prod/softprod/apps/openpype/LINUX/3.15/dependencies/click/core.py", line 760, in main
2023-05-30 16:16:51:  0: STDOUT:     _verify_python3_env()
2023-05-30 16:16:51:  0: STDOUT:   File "/prod/softprod/apps/openpype/LINUX/3.15/dependencies/click/_unicodefun.py", line 126, in _verify_python3_env
2023-05-30 16:16:51:  0: STDOUT:     raise RuntimeError(
2023-05-30 16:16:51:  0: STDOUT: RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment. Consult https://click.palletsprojects.com/python3/ for mitigation steps.
```
## Additional info
This error was produce on centos7.

## Testing notes:
1. Open an asset on maya
2. Add render creator 
3. Open publish interface and run
4. See the render publish on deadline (with centos7 worker)
